index.html

```diff
@@ -633,10 +633,23 @@ <h2>Core Data Model</h2>
 
       <p>
 The following sections outline core data model concepts, such as <a>claims</a>,
-<a>credentials</a>, and <a>presentations</a>, which form the foundation of this
+<a>credentials</a>, <a>presentations</a>, <a>verifiable credentials</a>, and
+<a>verifiable presentations</a>, which form the foundation of this
 specification.
       </p>
 
+      <p class="note"
+        title="Not every abstract concept is realized in implementations">
```

Comment:
I don't feel this language captures consensus, or is accessible / comprehensible enough... but It think it might be possible to get to where we need to be, if we refer to media types, and normative requirements as they related to the vocabulary and context... which are normative.

Reply:
Please provide some concrete language.

I will note that this section explicitly talks to a conceptual data model and not a serialization, which is where media types come in. Keep that in mind when proposing concrete text.

Reply:
Possible language:

This specification does not define `credential` or `presentation` as conforming documents, or provide media types for expressing them. 

This specification does define `verifiable credential` and `verifiable presentation` as conforming documents, and provides media types for expressing them. 

There is no testable difference between a `credential` and `verifiable credential`, however we assert that at least one securing mechanism and the details necessary to evaluate it MUST be present to call a conforming document a verifiable credential.